README.md

```diff
@@ -26,11 +26,18 @@ See the language-specific guides for further details on support and usage of the
 
 Further information on the Azure SQL binding for Azure Functions is also available in the [Azure Functions docs](https://docs.microsoft.com/azure/azure-functions/functions-bindings-azure-sql).
 
+## Supported SQL Server Versions
+
+This extension uses the [OPENJSON](https://learn.microsoft.com/sql/t-sql/functions/openjson-transact-sql) statement which requires SQL Server 2016 or higher and a database compatibility level of 130+. Older versions will currently fail to execute properly.
+
+For more information about options to upgrade versions see the [SQL Server end of support options](https://learn.microsoft.com/sql/sql-server/end-of-support/sql-server-end-of-support-overview) page.
```

Comment:
This feels a little verbose to me.

Maybe something like `For more information about version upgrade options`

Reply:
I'd prefer this whole statement gets omitted, it's not about a discussion of what versions of the engine are in support - the SQL extension library has a requirement based on its architecture.